### PR TITLE
Blog post on best practices for success with Groundlight detectors

### PR DESCRIPTION
**Pull request by paulina-positronix**
Screenshot 2023-12-19 at 7 46 35 AM
Screenshot 2023-12-19 at 7 48 45 AM
page3

### CODE REVIEW

**Comment by positavi** [comment]
why does a rare event have to be a "no" instead of a yes?

what about the labeler feedback mechanism ?

File: docs/blog/2023-12-15-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Best practices for best results with Groundlight
+description: How to be a gold-star Groundlight customer
```
Comment (positavi) [inline docs/blog/2023-12-15-best-practices.md:3]: better description?

File: docs/blog/2023-12-15-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+title: Best practices for best results with Groundlight
```
Comment (mjvogelsong) [inline docs/blog/2023-12-15-best-practices.md:2]: It sounds kinda weird to have two "best"s ...

Maybe "Best practices for Groundlight detectors" or "Getting the most out of your Groundlight detectors"?

**Review by mjvogelsong — APPROVED** [review]
Cool, looks good!

**Pull request merged**